ui/decredmaterial/editor.go

```diff
@@ -216,3 +225,19 @@ func (e Editor) handleEvents(gtx *layout.Context) {
 		e.editor.SetText("")
 	}
 }
+
+func (e *Editor) SetRequiredErrorText(txt string) {
+	e.requiredErrorText = txt
```

Comment:
Why do we need to set required error text? aren't all required message for input fields "field is required"?